pkg/virtctl/imageupload/imageupload.go

```diff
@@ -236,10 +249,28 @@ func parseArgs(args []string) error {
 	return nil
 }
 
+func validateDefaultInstancetypeArgs() error {
+	if defaultInstancetype == "" && defaultInstancetypeKind != "" {
```

Comment:
Is the default kind really mandatory or would a PVC/DV with only the default instancetype/preference label result in the admitter/mutators filling in the default cluster kind?

Reply:
I think you have this conditional the wrong way around. It's ensuring that `--default-instancetype` is always provided with `--default-instancetype-kind`, `--default-instancetype-kind` is entirely optional and our webhooks will default to the cluster type when submitted.

Reply:
Yeah, that was still too early. Sorry for the noise.